### PR TITLE
[SOFA-16] Fix bugs

### PR DESCRIPTION
**Pull request by vihan85**

### CODE REVIEW

File: src/home.html [hunk 1]
```diff
@@ -39,7 +39,9 @@
                         0967957072
                     </a>
                 </div>
-
+                <div class="site-header_logo-mobile">
```
Comment (huynhhuulocit) [inline src/home.html:42]: @vihan85 
Fix

File: src/js/app.js [hunk 2]
```diff
@@ -65,43 +64,18 @@ $(document).ready(function () {
...
-                    slidesToScroll: 1,
-                    infinite: true,
-                    dots: true,
-                    
-                    
-                  }
-                }
-            ]
-        });
+    //slick
```
Comment (huynhhuulocit) [inline src/js/app.js:67]: @vihan85 
test

**Review by huynhhuulocit — COMMENTED** [review]
@vihan85 
Fix and report

File: src/home.html (hunk 1 above)
Comment (vihan85) [reply src/home.html:42]: 6

File: src/js/app.js (hunk 2 above)
Comment (vihan85) [reply src/js/app.js:67]: 65

**Pull request merged**